src/com/softserve/academy/edu1/hw5/taskHW1/DaysInMonth.java

```diff
@@ -0,0 +1,51 @@
+package softserve.academy.edu1.hw5.taskHW1;
+
+import java.time.Month;
+import java.util.Scanner;
+
+public class DaysInMonth {
+    static final Scanner SCANNER = new Scanner(System.in);
+
+    public static boolean isLeapYear() {
```

Comment:
it would be better to separate this method to two:
String inputString(String message)
and
isEnterYes(String answer)